docs/source/app-dev/bindings-java/quickstart/template-root/daml/Tests/Helper.daml

```diff
@@ -0,0 +1,18 @@
+-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Tests.Helper where
+
+import Iou
+
+template Helper
+  with
+    party : Party
+  where
+    signatory party
+    controller party can
+      FetchIou : Iou
```

Comment:
Perhaps worth adding a comment here that we do this because `fetch` is not a Ledger API command.

Reply:
Good point, I've added a comment pointing to the relevant docs.daml.com as well.

Reply:
Should we consider adding `fetch` to the ledger API? cc @gerolf-da

Reply:
We added `queryContractId` to the DAML Script API in the meantime and used it here.

Reply:
@hurryabit: seems like an easy enough "surface area" change, but would need more spec on visibility, what other meta data would you want along with the LF Value itself, etc...